modules/covid_vaccine/app/services/covid_vaccine/v0/expanded_registration_csv_generator.rb

```diff
@@ -17,13 +17,19 @@ class ExpandedRegistrationCsvGenerator
         r.raw_form_data['preferred_facility']&.delete_prefix('vha_')
       end
 
+      birth_sex_proc = proc do |r|
+        next r.raw_form_data['birth_sex'][0] if %w[Male Female].include? r.raw_form_data['birth_sex']
```

Comment:
how about 

How about:

```suggestion
        { 'Male' => 'M', 'Female' => 'F' }.fetch(r.raw_form_data['birth_sex'][0].upcase, nil)
```

or 

```suggestion
        %w[M F]).find(r.raw_form_data['birth_sex'][0].upcase)
```

Reply:
might not even need the `upcase` if it's always one of those three values